php/api/controller/class-audit-posts-controller.php

```diff
@@ -139,13 +141,36 @@ public function register_routes() {
 			'schema' => array( $this, 'get_public_item_schema' ),
 		) );
 
-		register_rest_route( $this->namespace, '/' . $this->rest_base . '/' . static::PROJECT_PATTERN . '/' . static::SLUG_PATTERN, array(
+		register_rest_route( $this->namespace, '/' . $this->rest_base . '/' . static::PROJECT_PATTERN . '/' . static::PROJECT_TYPE_PATTERN, array(
 			'args'   => array(
-				'project' => array(
+				'project'      => array(
+					'description' => __( 'The users slug.' ),
+					'type'        => 'string',
+				),
+				'project_type' => array(
+					'description' => __( 'The project type: theme or plugin.' ),
+					'type'        => 'string',
+				),
+			),
+			array(
+				'methods'  => \WP_REST_Server::READABLE,
+				'callback' => array( $this, 'get_audits' ),
+				'args'     => $get_item_args,
+			),
+			'schema' => array( $this, 'get_public_item_schema' ),
+		) );
+
+		register_rest_route( $this->namespace, '/' . $this->rest_base . '/' . static::PROJECT_PATTERN . '/' . static::PROJECT_TYPE_PATTERN . '/' . static::SLUG_PATTERN, array(
+			'args'   => array(
+				'project'      => array(
 					'description' => __( 'The users slug.' ),
 					'type'        => 'string',
 				),
-				'slug'    => array(
+				'project_type' => array(
+					'description' => __( 'The project type: theme or plugin.' ),
+					'type'        => 'string',
+				),
+				'slug'         => array(
```

Comment:
Shouldn't this be `project_slug` and the description be plugin/theme agnostic.

Reply:
copied and pasted from what was there before... makes sense.